src/app/page.tsx

```diff
@@ -1,111 +1,13 @@
 import Image from 'next/image'
+import MarketingMaterials from '@/components/MarketingMaterials'
+import englishImage from '/Users/krishmalik/Documents/inreach-website/src/app/images/english-images.png'; 
```

Comment:
Please don't use absolute paths. This will fail in production. You need to use relative paths, i.e.
```
import englishImage from './images/english-images.png
```